docs/tutorials/api/datasets.ipynb

```diff
@@ -0,0 +1,877 @@
+{
```

Comment:
<p>saved to disk</p><p> </p><p>I would also mention that this saves the various <code>Maps</code>  as various HDUs  of a single FITS file. And that the <code>Maps</code>  are currently stored according to the gadf specifications for skymaps.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>